Gemfile

```diff
@@ -260,6 +260,10 @@ group :web_socket, :manageiq_default do
   gem "websocket-driver",               "~>0.6.3",           :require => false
 end
 
+group :appliance do
```

Comment:
Thinking this should go in the `ENV["APPLIANCE"]` section

Reply:
I originally had that, but at the moment I don't see much advantage.  `manageiq-appliance_console` should install without issue on dev and test environments and it reduces the differences in dependency resolution between dev, test and prod.

Reply:
that's fair...ultimately we discussed removing ENV["APPLIANCE"] in favor or group :appliance, so maybe this is the start of that.

Reply:
Ok, I wrapped it in `ENV["APPLIANCE"]` for now since dependencies are missing for `rbnacl`

Reply:
Looks like most our gems are mac and linux friendly.
As long as we are ok with all the `rspec` like gems being installed on the appliance, sounds like a good plan to drop the `ENV`